include/grpcpp/ext/server_metric_recorder.h

```diff
@@ -0,0 +1,83 @@
+//
+//
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+#ifndef GRPCPP_EXT_SERVER_METRIC_RECORDER_H
+#define GRPCPP_EXT_SERVER_METRIC_RECORDER_H
+
+#include <atomic>
+#include <memory>
+#include <string>
+
+#include "absl/strings/string_view.h"
+#include "absl/types/optional.h"
+
+namespace grpc_core {
+class Arena;
```

Comment:
This forward-declaration is not needed in this file.

Reply:
Done.